### DISCUSSION TITLE
How to pass more args to the various models?

### DISCUSSION THREAD

**Discussion by feltz916** [absadiki/subsai#107] [General]
Everything is working great for me through the docker image now using CUDA cores, however there are some arguments that I'd like to pass to the config that I cannot do outside of command line.

ie:
--max_words_per_line
--max_line_count
--max_line_width

Is there a simple way to add this functionality without having to go to cmd line?

**Comment by absadiki** [comment]
No unfortunately ..  even though I tried to expose as many arguments as I can to the webui. 
But I made it fairly easy to add new arguments, just modify the `config_schema` attribute in the model file and they will be available in the webui.
Or let me know which model you want to apply these argument to and I might add them as soon as I can.
